emcc.py

```diff
@@ -102,7 +102,7 @@
   'emscripten_scan_registers', 'emscripten_lazy_load_code',
   'emscripten_fiber_swap',
   'wasi_snapshot_preview1.fd_sync', '__wasi_fd_sync', '_emval_await',
-  'dlopen',
+  'dlopen', 'asyncjs__*'
```

Comment:
How about `__asyncjs_*`? (double underscore prefix indicates it is an internal method)

Reply:
Sounds good to me. I changed it a few times already and wasn't sure which one to stop on 😅 I guess my only concern is that it's not _really_ an internal method, but a user-defined function that we simply prefixed. But on the other hand, `__` would reduce chances of a name clash...